js/libs/keycloak-js/dist/keycloak.d.ts

```diff
@@ -531,8 +531,10 @@ declare class Keycloak {
 
 	/**
 	* Called when a AIA has been requested by the application.
+	* @param status the outcome of the required action
+	* @param action the name of the required action
 	*/
-	onActionUpdate?(status: 'success'|'cancelled'|'error'): void;
+	onActionUpdate?(status: 'success'|'cancelled'|'error', action?: string): void;
```

Comment:
Is `action` here a finite amount of options that are static in nature? If so can we list all the options explicitly as we do with `status` before it? If not, this comment can be ignored.

Reply:
The values for the `action` parameter are the provider aliases for required actions like `UPDATE_PASSWORD`, `CONFIGURE_TOTP`  etc. Some of those required action aliases are indeed static values (see: [org.keycloak.models.UserModel.RequiredAction](https://github.com/keycloak/keycloak/blob/main/server-spi/src/main/java/org/keycloak/models/UserModel.java#L233)  however, since users can define their own required actions with custom aliases, we cannot use a static list here.